lib/matplotlib/font_manager.py

```diff
@@ -947,6 +948,24 @@ def copy(self):
         return new
 
 
+class FontsInterface:
+    def __init__(self, ordered_family):
+        self.set_orderedfamily(ordered_family)
+
+    def set_orderedfamily(self, ordered_family):
+        self.families = []
+        self.filepaths = []
+        for key, value in ordered_family.items():
+            self.families.append(key)
+            self.filepaths.append(value)
+
+    def get_families(self):
```

Comment:
A lot of the matplotlib code is written in a typical Java/C++ style with setters and getters, but I don't think it's necessarily a good way to design Python code. Why not just use `x.families` and `x.filepaths` directly? If it's important that they are not modified after creation, make them tuples so that calling code cannot accidentally append or delete items.

Reply:
That makes sense, this class was designed on similar terms to other such public classes, but we don't really want a user to modify these after creation, so it makes sense to keep them as tuples (and possibly make them private?)

Reply:
Removed this class altogether now..